src/blog/2023/03/node-red-3.1-beta-available-flowforge-cloud.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Node-RED 3.1 Beta 2 available on FlowForge Cloud
+subtitle: Test drive the next version of Node-RED
+description: To allow testing of the next version of Node-RED, FlowForge Cloud made the 3.1-beta-2 release available.
+date: 2022-03-10
+authors: ["zeger-jan-van-de-weg"]
+tags:
+    - posts
+---
+
+We are excited to announce that Node-RED 3.1 Beta 2 is now available on
+FlowForge Cloud! By making this version available as beta release we hope users
+validate the beta release and provide feedback on the release before the beta
+badge is dropped.
+
+<!--more-->
+
+To help our users test their flows in this new version, FlowForge allows you to
+copy an application through the "Duplicate Project" button in the settings. By
+clicking this button the version of Node-RED can be changed to the Beta release.
+We'd appreciate any feedback on the new Node-RED version, especially if there's
+backward incompatible changes. This way we can improve the release before the
+new version of Node-RED is deployed to production.
+
+While testing if your flows work as expected, you might enjoy using the improved
+editor context menu through right-clicking on the workspace, try out the flow
+locking, or add images to flow descriptions.
```

Comment:
```suggestion
locking feature, and the ability to add images to flow descriptions.
```

Reply:
It is not clear to me how you can use the last two features? Are these menu items?